src/engine/guidance/post_processing.cpp

```diff
@@ -4,6 +4,8 @@
 #include "engine/guidance/toolkit.hpp"
 
 #include <boost/assert.hpp>
+#include <boost/range/algorithm_ext/erase.hpp>
+
```

Comment:
missing: utility, cstddef
also: vector is already included in the header (interface requires vectors as in/out types)